local_dev_scripts/device-deploy.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+source ./device-build.config
```

Comment:
This should be using a path relative to the current file (and not pwd), see files in script for the `DIR=....` command